Backpack-SwiftUI/Calendar/Classes/Core/CalendarMonthGrid.swift

```diff
@@ -0,0 +1,106 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+struct CalendarMonthGrid<
+    DayCell: View,
+    EmptyLeadingDayCell: View,
+    EmptyTrailingDayCell: View
+>: View {
+    let monthDate: Date
+    let calendar: Calendar
+    let validRange: ClosedRange<Date>
+
+    @ViewBuilder let dayCell: (Date) -> DayCell
+    @ViewBuilder let emptyLeadingDayCell: () -> EmptyLeadingDayCell
+    @ViewBuilder let emptyTrailingDayCell: () -> EmptyTrailingDayCell
+
+    private let daysInAWeek = 7
+    
+    var body: some View {
+        // Sunday is the first day of the week, so we need to offset the days to make Monday the first day
```

Comment:
I think you don't need the 1st line of this comment

Reply:
ahh good catch!